project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+        costs: *costs_ashp_ducted_SEER_16_9-5_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-5_HSPF
+      - &ducted_ASHP_EnergyStarCC_exist_backup_shared_ducts_80_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.5 HSPF, CCHP, Integrated Backup, 80% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_80pctAFUE_FO_furnace_apply_logic      
+        costs: *costs_ashp_ducted_SEER_16_9-5_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-5_HSPF
+      - &ducted_ASHP_EnergyStarCC_exist_backup_shared_ducts_92-5_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.5 HSPF, CCHP, Integrated Backup, 92.5% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic      
+        costs: *costs_ashp_ducted_SEER_16_9-5_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-5_HSPF
+
+
+  - upgrade_name: Higher efficiency ducted ASHP w/ exist backup  #also provides SEER 15 cooling measure profile
+    # Measure 3
+    options:
+      #ASHP
+      - &ducted_ASHP_HigherEfficiency_exist_backup_no_shared_ducts
+        option: HVAC Heating Efficiency|ASHP, SEER 20, 11 HSPF, CCHP, Separate Backup # same as base option
+        apply_logic: *dualfuel_ASHP_no_ducted_furnace_apply_logic
+        costs: &costs_ashp_ducted_SEER_20_11_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3907.01 # for HSPF 9
+            multiplier: Fixed (1)
+          - value: 155.17
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ashp_ducted_SEER_20_11_HSPF 15
+
+
+      - *HP_cooling_efficiency_ducted
+      - *100_percent_conditioned_ducted
+
+      - *ng_boiler_secondary_heating_type_fuel
+      - *propane_boiler_secondary_heating_type_fuel
+      - *fuel_oil_boiler_secondary_heating_type_fuel
+      - *secondary_boiler_72
+      - *secondary_boiler_76
+      - *secondary_boiler_80
+      - *secondary_boiler_82
+      - *secondary_boiler_85
+      - *secondary_boiler_90
+      - *secondary_boiler_95
+      - *secondary_boiler_96
+      - *ng_wallfloor_furnace_secondary_heating_type_fuel
+      - *propane_wallfloor_furnace_secondary_heating_type_fuel
+      - *fuel_oil_wallfloor_furnace_secondary_heating_type_fuel
+      - *secondary_wallfloor_furnace_60
+      - *secondary_wallfloor_furnace_68
+
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0     
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+
+ 
+
+#### ducted furnace ####
```

Comment:
Another request to clean up the comments. I expect leftmost column to be like, a new upgrade or something.